src/lib/config.js

```diff
@@ -29,9 +29,55 @@
  ******/
 const RC = require('rc')('ALS', require('../../config/default.json'))
 
+const getOrDefault = (value, defaultValue) => {
+  if (value === null || value === undefined) {
+    return defaultValue
+  }
+
+  return value
+}
+
 module.exports = {
   API_PORT: RC.API_PORT,
-  DATABASE_URI: RC.DATABASE_URI,
+  DATABASE: RC.DATABASE_URI || {
```

Comment:
I'm not so sure about this... if a data uri is supplied then it will ignore the config. does that seem correct?

Reply:
Yeah I also was not 100% sure about it. I was just thinking of a good way to keep everything backwards compatible, but perhaps this is unnecessary.